.github/workflows/conformance-eks-v1.11.yaml

```diff
@@ -240,6 +257,12 @@ jobs:
         run: |
           cilium connectivity test ${{ steps.vars.outputs.connectivity_test_defaults }}
 
+      - name: Run e2e tests
+        if: ${{ github.event.label.name == 'ci/eni-e2e' || needs.check_changes.outputs.eni_tested == 'true' }}
```

Comment:
I suppose the goal here is to have a label `ci/eni-e2e` that might be used to force run the Ginkgo tests even if there was no code change?